lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -2468,6 +2468,49 @@ static LogicalResult verifyHWStructCastOp(HWStructCastOp cast) {
   return success();
 }
 
+static LogicalResult verifyBitCastOp(BitCastOp cast) {
+  StringRef errorMsg;
+  // Infer the bitwidth of the type, if it can be computed. Else return false.
+  std::function<bool(FIRRTLType, size_t &)> getWidth =
```

Comment:
Please move this to a helper function in FIRRTLTypes.h/cpp

Reply:
Also, please don't name it "getFoo" since it returns a bool.  "isFoo" would make more sense.

Reply:
Added the function to return the width. 
A non zero width is valid, any other unknown or zero width is treated as error.